build/Dockerfile

```diff
@@ -3,7 +3,7 @@ LABEL version=0.0.5
 
```

Comment:
Please increment version to 0.0.6

We should then push a new image to foundationdb/foundationdb-build.  I think Alvin and I are the only ones with the credentials to do that right now, though.  So we'll need to then bump the `image:` in docker-compose.yaml, as a later PR.

... actually, I think Alvin and I are the only people with the password to be able to